packages/strapi-admin/admin/src/containers/AdminPage/index.js

```diff
@@ -222,7 +266,11 @@ export class AdminPage extends React.Component {
               <Route path="/plugins/:pluginId" component={PluginPage} />
               <Route path="/plugins" component={ComingSoonPage} />
               <Route path="/list-plugins" component={ListPluginsPage} exact />
-              <Route path="/marketplace" render={this.renderMarketPlace} exact />
+              <Route
```

Comment:
Why the other components are not formatted like this one?

Reply:
I believe it is due to its editorconfig with prettier